packages/app/plugin/src/ios/appDelegate.ts

```diff
@@ -22,15 +27,44 @@ export function modifyObjcAppDelegate(contents: string): string {
     return contents;
   }
 
+  if (
+    !methodInvocationLineMatcher.test(contents) &&
+    !fallbackInvocationLineMatcher.test(contents)
+  ) {
+    WarningAggregator.addWarningIOS(
+      '@react-native-firebase/app',
+      'The AppDelegate.m may be malfolmed. Skiping adding Firebase to it.',
```

Comment:
```suggestion
      'Unable to determine correct Firebase insertion point in AppDelegate.m. Skipping Firebase addition.',
```

Everyone loves to propose changes to user messages, vs working on code ;-)